pom.xml

```diff
@@ -52,6 +52,15 @@
       <artifactId>rest-assured</artifactId>
       <scope>test</scope>
     </dependency>
+      <dependency>
+          <groupId>io.quarkus.resteasy.reactive</groupId>
+          <artifactId>resteasy-reactive-common</artifactId>
+          <version>1.12.2.Final</version>
+      </dependency>
+    <dependency>
+      <groupId>io.rest-assured</groupId>
+      <artifactId>rest-assured</artifactId>
```

Comment:
As far as I can say it based on the changes the dependency is only used at test time. you can define a dependency in maven as a test dependency. By doing so the runtime code (src/main/java) will not depend on that depnencie. Please check that.